package/src/common/config.ts

```diff
@@ -35,14 +35,15 @@ export function readConfiguration(
   const productName = getEnv("QUARTO_NAME");
   version = version || getEnv("QUARTO_VERSION");
 
-  const execPath = Deno.execPath();
-  const root = join(execPath, "..", "..", "..", "..", "..");
-  const pkg = join(root, getEnv("QUARTO_PACKAGE_DIR"));
-  const dist = join(pkg, getEnv("QUARTO_DIST_DIR"));
-  const share = join(dist, getEnv("QUARTO_SHARE_DIR"));
-  const src = join(root, getEnv("QUARTO_SRC_DIR"));
-  const out = join(pkg, getEnv("QUARTO_OUT_DIR"));
-  const bin = join(dist, getEnv("QUARTO_BIN_DIR"));
+  const root = getEnv("QUARTO_ROOT");
+  const src = getEnv("QUARTO_SRC_DIR");
+
+  const pkg = getEnv("QUARTO_PACKAGE_DIR");
+  const out = getEnv("QUARTO_OUT_DIR");
+
+  const dist = getEnv("QUARTO_DIST_DIR");
+  const share = getEnv("QUARTO_SHARE_DIR");
```

Comment:
Here is at least one place where we need to respect `QUARTO_SHARE_PATH` (there may be others)